lib/webrick/httprequest.rb

```diff
@@ -488,22 +488,26 @@ def parse_uri(str, scheme="http")
       str.sub!(%r{\A/+}o, '/')
       uri = URI::parse(str)
       return uri if uri.absolute?
+      uri.scheme = @forwarded_proto || scheme
```

Comment:
Is there a reason this line was moved?

Reply:
Need the scheme to use URI.parse in my universe.

Reply:
Ignore that - not needed - will fix